dbux-graph-host/src/graph/controllers/GraphNode.js

```diff
@@ -36,12 +35,13 @@ export default class GraphNode extends HostComponentEndpoint {
 
   setOwnMode(mode) {
     this.setState({ mode });
-    if (this.owner.shouldLazyBuild && !this.owner.childrenBuilt) {
-      this.owner.buildChildNodes();
-    }
   }
 
   setMode(mode) {
+    if (mode !== GraphNodeMode.Collapsed && this.owner.childrenBuilt === false) {
```

Comment:
For booleans, we don't need `===` or `!==`. Unary operators `!!` and `!` instead.

Reply:
I want to ensure it that it is `false` rather than `undefined`. Some components like `RunNode`s also have their `GraphNode`s but they don't actually needs the lazy build system(I was using two properties `shouldLazyLoad` and `childrenBuilt` to maintain this). Maybe I should add some comment on it? Or how can I improve this?

Reply:
I see... In that case, just leave it for now. It's not ideal, but it gets the job done :)

Yeah, a comment would be great: Let everyone know it can be `true`, `false` or `undefined`.
You can use `|` (type union operator) for that: https://jsdoc.app/tags-type.html